### ISSUE TITLE
serial fftw3 test can't find symbols

### ISSUE THREAD

**Issue by brucellino** [SouthAfricaDigitalScience/fftw3-deploy#4]
since build 18, we can't compile serial applications of FFTW in the `deploy.sh` script.

We get the following errors : 

```
hello-world.cpp:(.text+0x1f2): undefined reference to `fftw_plan_dft_1d'
hello-world.cpp:(.text+0x217): undefined reference to `fftw_execute'
hello-world.cpp:(.text+0x235): undefined reference to `fftw_destroy_plan'
```

**Comment by brucellino** [comment]
I just tried this manually on a WN and the compilation and execution worked fine. It seems like there's an issue with the GCC modulefile, which is written wrong.

**Comment by brucellino** [comment]
Have submitted [gcc-deploy/3e175](https://github.com/SouthAfricaDigitalScience/gcc-deploy/commit/3e1756b1c60606c01651084abfda5aa97c38cb2c) to try to address this.

**Comment by brucellino** [comment]
This passed in build 24.

**Comment by brucellino** [comment]
fixed in d62ea5e9618
